apollo-router/src/services/subgraph_service.rs

```diff
@@ -975,8 +976,7 @@ async fn do_fetch(
     FetchError,
 > {
     let _active_request_guard = context.enter_active_request();
-    let response = client
-        .call(request)
+    let response = file_uploads::wrap_http_client_call(client, request)
```

Comment:
FYI this conflicts with https://github.com/apollographql/router/pull/4568 which was just merged, but that should not be a huge change, it keeps working on a hyper Body

Reply:
This got resolved